controllers/board-game-review-controllers.js

```diff
@@ -1,2 +1,18 @@
 const { request, response } = require("../app");
-const {} = require("../models/board-game-review-models");
\ No newline at end of file
+const { selectCategories } = require("../models/board-game-review-models");
+
+exports.getApi = (req, res) => {
+    res.status(200).send({ message: "ok" });
+  };
```

Comment:
again getApi shouldn't exist yet as it's a completely different ticket, please remove